charts/kasperleyn/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "kasperleyn.fullname" . }}
+  labels:
+{{ include "kasperleyn.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
+  selector:
```

Comment:
Why are we not using the same `selectorLabels` helper as above?

Reply:
Because Helm 2 doesn't `helm create` those.